utility/doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contribution guide
```

Comment:
I am not opposed to approving this as is, but the document seems to be very specific to the situation you are in right now: senior engineer on PNI working on E2E features.

I'm a bit afraid it could lead to confusion to external contributors but happy to get push back.

Reply:
Thank you for the review, makes perfect sense. I'll move this over to the feature path template doc.